cosmovisor/args.go

```diff
@@ -239,7 +239,7 @@ func (cfg *Config) UpgradeInfo() UpgradeInfo {
 	return cfg.currentUpgrade
 
 returnError:
-	fmt.Println("[cosmovisor], error reading", filename, err)
+	Logger.Error().Msgf("[cosmovisor], error reading %s %v", filename, err)
```

Comment:
nit: I would discourage this type of logging. Instead, use fields.

e.g.

```go
logger.With("module", "cosmovisor").Error().Msg("failed to read").Str("filename", filename)
```

Reply:
👍🏻